### DISCUSSION TITLE
stop drawer from hiding

### DISCUSSION THREAD

**Discussion by morningstarsabrina** [zauberzeug/nicegui#376] [Q&A] [answered]
Hello,

I'm having a problem with the drawer element being hidden when resizing the window.
here is my simple code
```py
from nicegui import ui

with ui.left_drawer():
    with ui.column():
        ui.button("button1")
        ui.button("button2")
ui.run(title='drawer', dark=True)
```
Here is the drawer
<a href="https://ibb.co/8rk738Q"><img src="https://i.ibb.co/K2nrZy8/1.png" alt="1" border="0"></a>

Here is the drawer being missing after resizing the window
<a href="https://ibb.co/p6Drz0T"><img src="https://i.ibb.co/JWLF7BT/2.png" alt="2" border="0"></a>

is there a way to make the drawer fixed ?

**Comment by falkoschindler** [comment, marked as answer]
You can write `ui.left_drawer().props('behavior=desktop')` to avoid the automatic switch between desktop and mobile mode.
See [Quasar's Drawer API](https://quasar.dev/layout/drawer#qdrawer-api) for details.

**Reply by morningstarsabrina** [reply]
thanks
